src/ServiceControl.AcceptanceTests/Recoverability/EditMessageHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ServiceBus.Management.AcceptanceTests.Recoverability
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+
+    public static class EditMessageHelper
+    {
+        public static List<KeyValuePair<string, string>> TryRestoreOriginalHeaderKeys(Dictionary<string, string> headers)
```

Comment:
this wouldn't be necessary if we fixed #1629 , e.g. as suggested in this PR: https://github.com/Particular/ServiceControl/pull/1631